docs/pipeline-configuration-files/specifications.mdx

```diff
@@ -276,22 +277,23 @@ beginning.
   identify the processor inside the pipeline. If multiple processors inside the
   pipeline use the same ID, provisioning will fail.
 
-### type
+### plugin
```

Comment:
We also introduced the field `condition` on a processor.

Reply:
that was the reason I opened this PR, how did I forget that 🥲